src/ocean/util/serialize/model/Traits.d

```diff
@@ -47,7 +47,7 @@ private struct _Dummy
 
 *******************************************************************************/
 
-template isSerializer(T)
+deprecated template isSerializer(T)
```

Comment:
Tricky bit here - the module is publicly imported via `package_.d` and can't be marked deprecated wtihout removing from the imports, which would be a breaking change on its own. I have no better idea how to fix that other than just kill it in next major.